### PR TITLE
handle case when Patroni returns that lag is unknown

### PR DESCRIPTION
**Pull request by FxKu**
In GitHub e2e tests we see the following error:
```
cannot unmarshal string into Go struct field ClusterMember.members.lag of type int
```

Turns out Patroni can return the string value `"unknown"` for replica lag - e.g. in case there's no leader. The operator must deal with this by using the `IntOrString` type by `intstr` lib.

Will take the chance to rename the field simply to `Lag`.

### CODE REVIEW

File: manifests/postgresql.crd.yaml [hunk 1]
```diff
@@ -462,11 +462,11 @@ spec:
                 type: integer
               standby:
                 type: object
-                required:
-                  - s3_wal_path
                 properties:
                   s3_wal_path:
                     type: string
+                  gs_wal_path:
+                    type: string
```
Comment (FxKu) [inline manifests/postgresql.crd.yaml:469]: leftover from #1446

File: pkg/util/util.go [hunk 2]
```diff
@@ -322,6 +323,20 @@ func testNil(values ...*int32) bool {
...
+// Convert int to IntOrString type
+func ToIntStr(val int) *intstr.IntOrString {
+	b := intstr.FromInt(val)
+	return &b
+}
+
+// Get int from IntOrString and return max int if string
+func IntFromIntStr(intOrStr intstr.IntOrString) int {
+	if intOrStr.Type == 1 {
+		return 32 << (^int(0) >> 32 & 1) // return max int
```
Comment (Jan-M) [inline pkg/util/util.go:335]: Is there no constant for max int in go?

Comment (FxKu) [reply pkg/util/util.go]: indeed: `math.MaxInt`

**Comment by FxKu** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

**Pull request merged**